rmw_connext_cpp/src/functions.cpp

```diff
@@ -1,3 +1,5 @@
+#include <rmw/types.h>
```

Comment:
Fixed, GitHub is too smart and doesn't detect that this comment is outdated, though :-P